modules/oe_list_pages_link_list_source/src/ContextualFiltersConfigurationBuilder.php

```diff
@@ -0,0 +1,330 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_list_pages_link_list_source;
+
+use Drupal\Component\Utility\NestedArray;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\oe_list_pages\FilterConfigurationFormBuilderBase;
+use Drupal\oe_list_pages\ListPresetFilter;
+use Drupal\oe_list_pages\ListSourceInterface;
+use Drupal\oe_list_pages\Plugin\facets\widget\MultiselectWidget;
+
+/**
+ * Builds the list page source contextual filters form elements.
+ */
+class ContextualFiltersConfigurationBuilder extends FilterConfigurationFormBuilderBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getAjaxWrapperId(array $form): string {
```

Comment:
We could maybe make a base function for this using getFilterType to generate the id but not required.